src/storage.rs

```diff
@@ -20,10 +21,11 @@ pub trait HasIndex {
     fn get_index() -> Option<Self::Index>;
 }
 
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum TypeStorer {
     Redact(redact::RedactStorer),
     Mongo(mongodb::MongoStorer),
+    GoogleCloud(google_cloud_storage::GoogleCloudStorer),
```

Comment:
can we rename the module to just `google` or `gcs` for simplicity/lack of underscores sake?

Reply:
renamed to gcs